<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMaintainLog</class>
 <widget class="QDialog" name="QMaintainLog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1760</width>
    <height>930</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QMaintainLog</string>
  </property>
  <widget class="QTableView" name="tableView_maintain_date">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>1660</width>
     <height>269</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#tableView_maintain_date QScrollBar::handle:vertical  {
    margin-top: 46px;
}

#tableView_maintain_date QScrollBar::sub-line:vertical  {
    height: 43px; 
}</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>0</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>0</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>45</number>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>429</y>
     <width>1660</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>详细信息</string>
   </property>
   <property name="bks" stdset="0">
    <string>gray</string>
   </property>
   <property name="bordersL" stdset="0">
    <string>left</string>
   </property>
   <property name="bordersT" stdset="0">
    <string>top</string>
   </property>
   <property name="bordersR" stdset="0">
    <string>right</string>
   </property>
   <property name="paddings_left" stdset="0">
    <string>l_10</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_maintain_result">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>474</y>
     <width>1660</width>
     <height>332</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QScrollBar::handle:vertical  {
    margin: 2px;
}
QScrollBar:vertical  {
    width: 13px;
}
</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="html">
    <string notr="true">&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QFrame" name="devListFrame">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>90</y>
     <width>1126</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>0</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="line_h">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>1660</width>
     <height>1</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string>solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_v">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>1</width>
     <height>269</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string>solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_v_2">
   <property name="geometry">
    <rect>
     <x>1709</x>
     <y>150</y>
     <width>1</width>
     <height>269</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string>solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_h_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>418</y>
     <width>1660</width>
     <height>1</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string>solid</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>471</width>
     <height>40</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>日期选择</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_year">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>40</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>2022</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_mon">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>40</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>1月</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2月</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3月</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4月</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5月</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6月</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7月</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8月</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9月</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10月</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>11月</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12月</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>47</x>
     <y>833</y>
     <width>1666</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>14</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="last_mon_btn">
      <property name="minimumSize">
       <size>
        <width>116</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>上一月</string>
      </property>
      <property name="buttons" stdset="0">
       <string>btn_dlg</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="next_mon_btn">
      <property name="minimumSize">
       <size>
        <width>116</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>下一月</string>
      </property>
      <property name="buttons" stdset="0">
       <string>btn_dlg</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="print_maintain_btn">
      <property name="minimumSize">
       <size>
        <width>116</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>打印</string>
      </property>
      <property name="buttons" stdset="0">
       <string>btn_dlg</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="output_btn">
      <property name="minimumSize">
       <size>
        <width>116</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>导出</string>
      </property>
      <property name="buttons" stdset="0">
       <string>btn_dlg</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="close_btn">
      <property name="minimumSize">
       <size>
        <width>116</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>关闭</string>
      </property>
      <property name="buttons" stdset="0">
       <string>btn_dlg</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>tableView_maintain_date</zorder>
  <zorder>label_3</zorder>
  <zorder>textEdit_maintain_result</zorder>
  <zorder>devListFrame</zorder>
  <zorder>line_h</zorder>
  <zorder>line_v</zorder>
  <zorder>line_v_2</zorder>
  <zorder>widget</zorder>
  <zorder>widget_2</zorder>
  <zorder>line_h_2</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
